src/json.h

```diff
@@ -234,7 +238,11 @@ class Array : public Value, public std::vector<std::unique_ptr<Value>> {
 class Object : public Value, public std::map<std::string, std::unique_ptr<Value>> {
  public:
   Object() {}
+  // Construct an object with the specified fields.
+  // nullptr values will be discarded.
```

Comment:
Maybe be more explicit:

// key/value entries where the value is nullptr will be discarded.